app/decorators/session_decorator.rb

```diff
@@ -20,4 +20,12 @@ def idv_hardfail4_partial
   end
 
   def logo_partial; end
+
+  def timeout_flash_text
+    I18n.t('notices.session_cleared', minutes: Figaro.env.session_timeout_in_minutes)
+  end
+
+  def sp_name
+    nil
```

Comment:
might prefer `def sp_name; end` for consistency with other methods that return `nil` in this class